fdbrpc/FlowTests.actor.cpp

```diff
@@ -1271,3 +1272,115 @@ TEST_CASE("/flow/DeterministicRandom/SignedOverflow") {
 	       std::numeric_limits<int64_t>::max() - 1);
 	return Void();
 }
+
+class PrivateKeyRangeTestImpl : public PrivateKeyRangeBaseImpl {
+public:
+	explicit PrivateKeyRangeTestImpl(KeyRef start, KeyRef end, std::string prefix, int size) : PrivateKeyRangeBaseImpl(start, end) {
+		this->prefix = prefix;
```

Comment:
Either std::move prefix here or take it by const reference. Also, use a [member initializer list](https://en.cppreference.com/w/cpp/language/initializer_list)

Reply:
Done. Thanks.